### ISSUE TITLE
Error: system message is required for full chat completion on summarize request

### ISSUE THREAD

**Issue by tomlue** [bakks/butterfish#15]
```
> butterfish summarize README.md
Loaded 11 prompts from ...
Summarizing README.md
Error: system message is required for full chat completion
```

**Comment by bakks** [comment]
Heyo, this was a regression in the last release, fixing now

**Comment by bakks** [comment]
This should be fixed in [v0.1.7](https://github.com/bakks/butterfish/releases/tag/v0.1.7), just released.

**Comment by tomlue** [comment]
can confirm new version is working for me.
